### ISSUE TITLE
Move from master to gh-pages?

### ISSUE THREAD

**Issue by davelab6** [daltonmaag/scope-one#11]
Following https://github.com/daltonmaag/scope-one/pull/10 I propose to move the leading branch from master to gh-pages, which can be done in https://github.com/daltonmaag/scope-one/settings and then merging master into gh-pages and then removing master locally and pushing its removal.

**Comment by anthrotype** [comment]
so are you saying the repo should only have one branch called `gh-pages`?
I don't know if that's a good idea. I'd still like to keep a `master` branch called `"master"`.

**Comment by davelab6** [comment]
Yes, that's my idea. However, you can also set up Travis to build, test,
and push the build result to the gh-pages branch. Would you prefer that? :)

**Comment by djdm** [comment]
> However, you can also set up Travis to build, test,
and push the build result to the gh-pages branch. Would you prefer that? :)

We were thinking of doing that instead.
